practice-two/src/css/main.css

```diff
@@ -34,6 +34,43 @@
   display: none;
 }
 
+.best-location-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 32px;
+  .carousel-button {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+  }
+  .previous-btn {
+    width: 64px;
+    height: 64px;
+    background: #fff;
+    border-radius: 100%;
+    border: 1px solid rgba(25, 24, 37, 0.5);
+  }
+  .next-btn {
+    width: 64px;
```

Comment:
pre and next buttons should have the same styles. They only differ in its content (icon).

Reply:
i will check it